modules/sdk-coin-avaxc/test/unit/avaxc.ts

```diff
@@ -619,4 +619,54 @@ describe('Avalanche C-Chain', function () {
         .should.be.rejectedWith('coin in txPrebuild did not match that in txParams supplied by client');
     });
   });
+
+  // TODO(BG-56136): move to modules/bitgo/test/v2/integration/coins/avaxc.ts
+  describe('Recovery', function () {
+    it('should error when the backup key is unfunded (cannot pay gas)', async function () {
+      await tavaxCoin
+        .recover({
+          userKey:
+            '{"iv":"ntd9/urFjryqxd4rzREB2Q==","v":1,"iter":10000,"ks":256,"ts":64,"mode"\n' +
+            ':"ccm","adata":"","cipher":"aes","salt":"LTqZ47b1BwE=","ct":"JSbJIBTkHoKR3L\n' +
+            'oT2QTkDx3X1OBIPxiSL6WoMiIrKA+aKTgmutXnWC2GTEIyfbLeajw6D2UZ+U0Y8viv7mgITgSz1\n' +
+            'u9Gdj97Btm8WsZ0e+KmsbdB/gYucCZoPUZCFqG4bEkdfZ8ZvDI9XvVv4xPzNb/AoSijosA="}',
+          backupKey:
+            '{"iv":"Axs+G9gsZ5PENUHx1YY5cg==","v":1,"iter":10000,"ks":256,"ts":64,"mode"\n' +
+            ':"ccm","adata":"","cipher":"aes","salt":"awQshUvFi7Q=","ct":"sWQ0bHmruUTI8C\n' +
+            'lwGneHObdNfo3WQ/mrz3p84Fo07HgizvgLd+E3wFA3Z1LRbHozRjfstV/qJMRqrFvEgKOcG+SKd\n' +
+            'gx6BbmXWfKhFHEerSYluBgU5OrXMfOkbExnMywEWrCKEvoNL+wyNHoRaMNbbDogo36J8PE="}',
+          walletContractAddress: '0x22c1ab44371985e49294d1a40e92c8ad00f5be8e',
+          // walletPassphrase: TestBitGo.V2.TEST_RECOVERY_PASSCODE,
+          walletPassphrase: 'Ghghjkg!455544llll',
```

Comment:
I don't love this

Reply:
This is supposed to be an integration test. There's a similar test in `modules/bitgo/test/v2/integration/coins/eth.ts` but it's broken at the `TestBitGo` import, so I decided to add it here [temporarily](https://github.com/BitGo/BitGoJS/pull/2699/files#diff-99b0fe147a9b6fb5dcc112de929157c239816a85dcbc3acb146faead047d7f69R623), since it's blocking WRW work.